gemrb/plugins/CREImporter/CREImporter.cpp

```diff
@@ -428,6 +428,12 @@ static void GetKitSpell(const ResRef& tableRef, std::vector<SpellEntry*>& list)
 			index = FindSpell(spellRef, splList);
 			assert (index != TableMgr::npos);
 		}
+
+		// IWD2 listdomn gives us dups here
```

Comment:
spwi903 is the only one, at level 8 for Oghmatiques, at level 9 for CTE.
Anyway, wouldn't this way one of them not get the spell? I think the correct thing to do is to rerun an amended loop below if the entry already exists (overwrite values if they are 0). This is basically a data problem, the two adjacent lines should have been one.

Reply:
The `SpellEntry` uses a vector for all this, so overwriting using the existing interface won't work. Will look at this tomorrow.

Reply:
Minor code cleanup now.